Core/Src/main.c

```diff
@@ -21,10 +21,13 @@ int main(void)
 
 	UB_VGA_Screen_Init(); // Init VGA-Screen
 
-	UB_VGA_FillScreen(VGA_COL_BLUE);
+	UB_VGA_FillScreen(VGA_COL_YELLOW);
 
   while(1)
   {
-
+	  for(uint8_t i=20; i<100; i++)
+	  {
+		  UB_VGA_SetPixel(160, i, VGA_COL_RED);

```

Comment:
Geen nummers midden in code, Define gebruiken
![DefineGoodKassandraLeeGIF](https://user-images.githubusercontent.com/46404391/164496762-60a18289-095b-44ec-8236-512ca6fb122a.gif)

Reply:
Defines aangemaakt.

Reply:
👍

Reply:
![BoratBoratVeryNiceGIF](https://user-images.githubusercontent.com/46404391/164504182-59ccc0fa-64d1-405e-96a4-062d23a4b1cd.gif)